client/app/admin-dashboard/page.jsx

```diff
@@ -1,9 +1,14 @@
-import AdminDashboard from "../../components/AdminDashboard";
+import { Suspense } from "react";
+import { Typography } from "@mui/material";
+import ReservationTable from "../../components/admin_dashboard/ReservationTable";
 
 export default function AdminDashboardView() {
   return (
     <div>
-      <AdminDashboard />
+      <Typography variant="h1">Reservations:</Typography>
+      <Suspense fallback={<p>Loading...</p>}>
```

Comment:
There are some nice built in loading indicators:
https://mui.com/material-ui/react-progress/